project/UI/wrapper.js

```diff
@@ -0,0 +1,110 @@
+;var module =(function(){
+
+    var userName = "Kurch_Dmitry";
+    var posts = control.getPhotoPosts();
+    var postList = document.querySelector(".divMain");
+
+    function createPhotoPost(postConfig){
+        var post = document.getElementById("templatePost").cloneNode(true);
+        post.className = "none";
+        post.id = null;
+
+        post.querySelector(".divNamePost").innerText = postConfig["author"];
```

Comment:
Do you know, that you do ``postConfig.author`` instead of postConfig["author"] ?